.gitignore

```diff
@@ -7,3 +7,4 @@ build/
 **/__pycache__/*
 *.vcd
 **/.hypothesis/
+tests
```

Comment:
Why ignore tests?

Reply:
The `tests` file contains the output of `redo tests.do`, i.e. whatever `pytest` prints when run. (It was empty before.)

All the `*_tests.py` files still get included -- this isn't a glob.